distage/distage-framework-docker/src/main/scala/izumi/distage/docker/Docker.scala

```diff
@@ -142,14 +191,25 @@ object Docker {
   final case class ClientConfig(
     readTimeoutMs: Int = 60000,
     connectTimeoutMs: Int = 1000,
-    allowReuse: Boolean = !DockerSupportProperties.`izumi.distage.docker.reuse-disable`.asBoolean(false),
+    globalReusePolicy: GlobalDockerReusePolicy = ClientConfig.parseReusePolicy(),
     useRemote: Boolean = false,
     useRegistry: Boolean = false,
     remote: Option[RemoteDockerConfig] = None,
     registry: Option[DockerRegistryConfig] = None,
   )
+
   object ClientConfig {
     implicit val distageConfigReader: DIConfigReader[ClientConfig] = DIConfigReader.derived
+    def parseReusePolicy(): GlobalDockerReusePolicy = {
+      DockerSupportProperties.`izumi.distage.docker.reuse`.strValue() match {
+        case Some("disabled") =>
```

Comment:
@pshirshov Should probably support "false" here for backward compatibility. Also, documentation mentions this and is now out of date.